synapse/swarm/runtime.py

```diff
@@ -59,11 +59,17 @@ def _getTufosByFrom(self, by, prop, valu=None, limit=None, fromtag=None):
 
     def _stormOperLift(self, query, oper):
 
```

Comment:
Some things differ such as query limit handling and # of acceptable args which is why this caught my eye.  Don't know if that is intentional or not.